src/server/debug.rs

```diff
@@ -1602,11 +1612,30 @@ mod tests {
     }
 
     impl Debugger {
-        fn set_store_id(&self, store_id: u64) {
-            let mut ident = StoreIdent::default();
-            ident.set_store_id(store_id);
+        fn get_store_ident(&self) -> Result<StoreIdent> {
             let db = &self.engines.kv;
-            db.put_msg(keys::STORE_IDENT_KEY, &ident).unwrap();
+            db.get_msg::<StoreIdent>(keys::STORE_IDENT_KEY)
+                .map_err(|e| box_err!(e))
+                .and_then(|ident| match ident {
+                    Some(ident) => Ok(ident),
+                    None => Ok(StoreIdent::default()),
+                })
+        }
+
+        fn set_store_id(&self, store_id: u64) {
```

Comment:
oh, it is strange to provide a dangerous set function here, especially for changing store and cluster ID. 

if you want to do this in test, you can get the debug kv engine and change the data directly like other tests do.

Reply:
These set functions are provided in the test mod. The `set_store_id` function is also used by other tests like `test_tombstone_regions`, `test_tombstone_regions_by_id`. Should I remove them and change the data from kv engine directly?

Reply:
I think the set functions are safe since they are only provided in the test mod. @siddontang PTAL